Need to #include <locale.h> 
as LC_ALL wasn't being found otherwise.

--- getopt-1.1.5/getopt.c.orig	2012-11-14 07:57:53.606143868 -0800
+++ getopt-1.1.5/getopt.c	2012-11-14 08:03:26.644112033 -0800
@@ -58,6 +58,7 @@
 #include <string.h>
 #include <unistd.h>
 #include <ctype.h>
+#include <locale.h>
 
 #if LIBCGETOPT
 #include <getopt.h>
